verdict.h

```diff
@@ -116,585 +122,585 @@ using namespace VERDICT_NAMESPACE;
 
 namespace VERDICT_NAMESPACE
 {
-const double VERDICT_DBL_MIN = 1.0E-30;
-const double VERDICT_DBL_MAX = 1.0E+30;
-const double VERDICT_PI = 3.1415926535897932384626;
+static constexpr double VERDICT_DBL_MIN = 1.0E-30;
+static constexpr double VERDICT_DBL_MAX = 1.0E+30;
+static constexpr double VERDICT_PI = 3.1415926535897932384626;
 
 /* quality functions for hex elements */
 
 //! Calculates hex edge ratio metric.
 /**  Hmax / Hmin where Hmax and Hmin are respectively the maximum and the
      minimum edge lengths */
-VERDICT_EXPORT double hex_edge_ratio(int num_nodes, const double coordinates[][3]);
+VERDICT_EXPORT VERDICT_HOST_DEVICE double hex_edge_ratio(int num_nodes, const double coordinates[][3]);
```

Comment:
I could fold in VERDICT_HOST_DEVICE into the definition of VERDICT_EXPORT if desired.